python/cac_tripplanner/destinations/models.py

```diff
@@ -90,6 +91,7 @@ class Meta:
                                    help_text='The large image. Will be displayed at 680x400.')
     published = models.BooleanField(default=False)
     priority = models.IntegerField(default=9999, null=False)
+    accessible = models.BooleanField(default=False, help_text='Is it ADA accessible?')
```

Comment:
This duplication makes me wonder if these two should be children of a base class (`Attraction`??).  They share 8 properties ([`name`, `website_url`, `description`, `published`, `priority`, `image`, `wide_image`, `accessible`]) and have 6 and 3 of their own, respectively ([`city`, `state`, `zip`, `address`, `point`, `categories`] for `Destination`, [`start_date`, `end_date`, `destination`] for `Event`).

The activities created for #917 will also be going on both.

Reply:
Awesome, looks great :circus_tent: